### PR TITLE
Implementation of login page design

### PR DESCRIPTION
**Pull request by Carreau**
Looks like that for now

capture d ecran 2014-10-01 a 16 44 46

Which not yet exactly what @awantulok  designed on https://github.com/jupyter/design/pull/4

Feel free to continue/use the PR this is mostly a test.

### CODE REVIEW

**Comment by Carreau** [comment]
6 month to rebase, but done.

**Comment by Carreau** [comment]
And move the error login message.

capture d ecran 2015-03-03 a 16 53 24

**Comment by minrk** [comment]
While this is probably an improvement for the login page, the effects of changing `brand-primary` to orange everywhere are not good:

screen shot 2015-03-04 at 14 20 09

screen shot 2015-03-04 at 14 20 02

I don't think orange is a good primary action color. Maybe special case the login button?

**Comment by Carreau** [comment]
capture d ecran 2015-03-06 a 12 25 42


Done, login screen stays, centered now also. Color closer to logo. Apply only to login page.

**Comment by Carreau** [comment]
We can also have transparent logo in the back (just to try how it looks)

capture d ecran 2015-03-06 a 12 38 59

File: share/jupyter/hub/static/less/login.less [hunk 1]
```diff
@@ -27,7 +29,65 @@
         margin-top: -1px;
     }
     
-    button[type=submit] {
+    input[type=submit] {
         margin-top: 16px;
+        background-color: @login-primary;
+        background: linear-gradient(to bottom,  @login-primary 0%, @login-secondary 100%); /* W3C */
```
Comment (minrk) [inline share/jupyter/hub/static/less/login.less]: can you get rid of the gradients? They don't fit in well with the otherwise flat colors.

Comment (minrk) [inline share/jupyter/hub/static/less/login.less]: Actually, I think the gradient on the header is okay. But definitely no gradient on the button.

**Pull request merged**